ee/tabby-webserver/src/handler.rs

```diff
@@ -8,61 +8,86 @@ use axum::{
 };
 use hyper::{Body, StatusCode};
 use juniper_axum::{graphiql, graphql, playground};
-use tabby_common::api::{code::CodeSearch, event::RawEventLogger, server_setting::ServerSetting};
+use tabby_common::api::{
+    code::CodeSearch,
+    event::{EventLogger, RawEventLogger},
+    server_setting::ServerSetting,
+};
+use tabby_db::DbConn;
```

Comment:
@wsxiaoys This needs to import DbConn unless we want to move it to a service, but I don't think that would make sense. Could we add an exception?